ApiClient.pqm

```diff
@@ -21,7 +21,15 @@ let
             queryParamsNonNull = Record.RemoveFields(combinedQueryParams, nullValueFields),
             queryString = Uri.BuildQueryString(queryParamsNonNull),
             apiUrlWithQueryParams = if Text.Length(queryString) > 0 then apiUrl & "?" & queryString else apiUrl,
-            parsedResponse = Json.Document(Web.Contents(apiUrlWithQueryParams, [Headers = headers]))
+            response = Web.Contents(
+                apiUrlWithQueryParams, [Headers = headers, ManualStatusHandling = {400, 401, 403, 404, 422, 500}]
```

Comment:
I think we should handle `503 Service Unavailable` as well, because that also happens occasionally.

```suggestion
                apiUrlWithQueryParams, [Headers = headers, ManualStatusHandling = {400, 401, 403, 404, 422, 500, 503}]
```